WikiDrips/WikiTableViewCell.swift

```diff
@@ -0,0 +1,27 @@
+//
+//  WikiTableViewCell.swift
+//  WikiDrips
+//
+//  Created by Bart Cullimore on 4/1/17.
+//  Copyright © 2017 Bart Cullimore. All rights reserved.
+//
+
+import UIKit
+
+class WikiTableViewCell: UITableViewCell {
+    
+    var wikiDoc: WikiDoc? {
```

Comment:
In the Model-View-Controller (MVC) pattern we usually don't pass models (i.e. `WikiDoc`) to views `WikiTableViewCell`. Setting the cell properties based on model object is usually done at Controller level.